K2Bridge.Tests.UnitTests/K2Bridge.Tests.UnitTests.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <CodeAnalysisRuleSet>CodeAnalysis.ruleset</CodeAnalysisRuleSet>
     <IsPackable>false</IsPackable>
+    <LangVersion>8</LangVersion>
```

Comment:
what is this

Reply:
Calms compiler because of the use of 'using' mid-text